backend/server.go

```diff
@@ -183,7 +213,7 @@ func StartGame(game *gamelogic.Game, responseWriter http.ResponseWriter, request
 
 	go game.ExecuteNight()
 
-	fmt.Fprint(responseWriter, "The Game's Afoot")
+	fmt.Fprint(responseWriter, "{}")
```

Comment:
Why do you not want success here?  Also instead of {} just remove the fprintln

Reply:
By far the main reason is convenience for the frontend. On the frontend it's convenient to have a guarantee that every endpoint returns json so i can call l `JSON.parse` indiscriminately. On another branch I have a function that looks something like this
```typescript
export async function req<T>(path: string): Promise<T> {
    resp = await fetch(url);
    const text = resp.text()
    const respJson = JSON.parse(text);
    return respJson;
}
```

If I returned the empty string or a success mesage then i'd get an error on that JSON.parse line. I could have the fronted figure out on the fly whether `text` was JSON, but then the return type of `req` would get complicated. That includes if the endpoint returned the empty string (i.e. not calling fprintln at all). JSON.parse would break.

Regarding the success message, i'm not opposed to returning `{ msg: "The game's Afoot"}`. But I think a success message is redundate with a 200 http status code.

Reply:
An easy way to do that is to change the return type of the APIFunctionHandler type we declared to require to return a map. Then the API will just add that as a JSON MSG.

Reply:
will do in a follow up pr